db/migrate/201502021111_add_source_type_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSourceTypeToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :source_type, :string, default: 'DB'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.